ex3/index.js

```diff
@@ -0,0 +1,40 @@
+const dayOfTheWeek = function (key) {
+  tempKey = parseInt(key);
```

Comment:
```suggestion
  const tempKey = parseInt(key);
```

o variabila trebuie sa fie delcarata fie cu `const` fie cu `let`